Kitodo/pom.xml

```diff
@@ -289,15 +268,19 @@
                 </exclusion>
             </exclusions>
         </dependency>
-
         <!-- https://mvnrepository.com/artifact/org.apache.commons/commons-email -->
+        <dependency>
```

Comment:
Comment above did not fit to this dependency as this comment belong to the next dependency.